### PR TITLE
API create stage

### PR DESCRIPTION
**Pull request by manhnv-1320**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/71184)

## HOW
create api create stage using apiview
## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by manhnv-1320** [comment]
ready

File: api/serializers.py [hunk 1]
```diff
@@ -82,3 +84,43 @@ def update(self, instance, validated_data):
...
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+
+        representation["user"] = UserStage.objects.values("user_id").get(
+            stage=instance, role=constants.STAGE_OWNER
+        )
+        return representation
+
+    def validate(self, data):
```
Comment (quocquann) [inline api/serializers.py:104]: tách hàm validate

**Review by quocquann — APPROVED** [review]

**Comment by huongnt-2545** [comment]
conflicted

**Comment by manhnv-1320** [comment]
ready

File: api/serializers.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 from rest_framework import serializers
 from rest_framework.validators import UniqueValidator
 
+from app.models import Stage, UserProject, UserStage
+from app.utils import constants
 from app.utils.helpers import check_token
 from app.models import Project
```
Comment (huongnt-2545) [inline api/serializers.py:11]: gộp với dòng 8

**Comment by manhnv-1320** [comment]
ready

**Pull request merged**